<!--
  Test the dynamic reconfiguration of the stamped_cmd_vel_mux. Run this after running the standalone launcher.
 -->
<launch>
  <node name="$(anon dynparam)" pkg="dynamic_reconfigure" type="dynparam" args="set_from_parameters nodelet_manager">
    <param name="yaml_cfg_file" type="string" value="$(find stamped_cmd_vel_mux)/param/reconfigure.yaml" />
  </node>
</launch>
